api/leaderboard.go

```diff
@@ -749,18 +723,12 @@ func (app *App) GetMembers(ctx context.Context, req *api.GetMembersRequest) (*ap
 	if tenantID != nil {
 		result, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```

Comment:
Instead of having line 731, what about this?

```suggestion
		members, err := app.Enricher.Enrich(ctx, tenantID[0], req.LeaderboardId, members)
```